cogs/customreactions.py

```diff
@@ -0,0 +1,1111 @@
+                                raise asyncio.TimeoutError
+
+                except asyncio.TimeoutError:
+                    if proposals:
+                        title = 'The modification of the custom reaction proposal timed out. ' \
+                                'Returning to list of reaction proposals...'
+                    else:
+                        title = 'The modification of the custom reaction timed out. ' \
+                                'Returning to list of current reactions...'
+                    embed = discord.Embed(title=title)
+                    await message.edit(embed=embed)
+                    conn.close()
+                    time.sleep(5)
+                    return
+
+                if response is None:
+                    return
+                elif response.lower() == "stop":
+                    await leave(message)
+                    return
+
+                t = (response, custom_react_id)
+                c.execute(
+                    'UPDATE CustomReactions SET Response = ? WHERE CustomReactionID = ?',
+                    t)
+                conn.commit()
+                self.rebuild_reaction_list()
+                self.rebuild_proposal_list()
+                if proposals:
+                    title = "Response successfully modified! Returning to list of reaction proposals..."
+                else:
+                    title = "Response successfully modified! Returning to list of current reactions..."
+                embed = discord.Embed(title=title)
+                embed.set_footer(text="Modified by {}.".format(user),
+                                 icon_url=user.avatar_url)
+                await message.edit(embed=embed)
+                conn.close()
+                time.sleep(5)
+
+            if reaction.emoji == "3️⃣":
+                embed = discord.Embed(title="Loading...")
+                await message.edit(embed=embed)
+                if proposals:
+                    title = "Modify a custom reaction proposal. React with the option you want"
+                    footer = "{} is currently modifying a custom reaction proposal. \n".format(
+                        user)
+                else:
+                    title = "Modify a custom reaction. React with the option you want"
+                    footer = "{} is currently modifying a custom reaction. \n".format(
+                        user)
+                description = "Should the message that calls the reaction be deleted?\n" \
+                              "0️⃣ No\n" \
+                              "1️⃣ Yes"
+                embed = discord.Embed(title=title, description=description)
+                embed.set_footer(text=footer, icon_url=user.avatar_url)
+                current_options.clear()
+                await message.clear_reactions()
+                current_options.extend(["0️⃣", "1️⃣", "⏹"])
+                await message.add_reaction("0️⃣")
+                await message.add_reaction("1️⃣")
```

Comment:
if we have the number tuple declared globally this could be done by taking a slice of the first 2 elements

Reply:
Resolved in 7d16cb6